user/details/style.css

```diff
@@ -277,4 +279,126 @@ footer p a {
     width: 20px;
     height: 20px;
   }
+
+  .tooltiptext::after {
+    left: 50%;
+  }
+}
+
+.user-pr {
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 1rem 0 1rem;
+}
+
+.pr-title {
+  font-size: 1.5rem;
+  text-align: center;
+}
+
+.pr-details {
+  display: flex;
+  flex-direction: row;
+  margin-top: 1rem;
+}
+
+.colon {
+  margin: 0;
+  padding: 0;
+  width: auto;
+}
+
+.pr-details-table {
+  border-collapse: collapse;
+  width: 100%;
+  margin-top: 1rem;
+}
+
+.pr-details-row:nth-child(even) {
+  background-color: var(--loader-gray-color);
+}
+
+.pr-details-head {
+  width: 50%;
+  padding: 8px 10px;
+}
+
+.pr-details-data,
+.pr-details-data-updated,
+.pr-details-data-created {
+  width: 50%;
+  position: relative;
+  padding: 0 10px;
+}
+
+.btn-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem;
+}
+
+.pr-view-btn {
+  padding: 0.5rem 1rem;
+  border-radius: 5px;
```

Comment:
please use `rem` instead of pixels and try to maintain consistency by using multiple of 4 this makes the design look good without much effort.  

4px (0.25rem), 8px(0.5rem), 12rem (0.75rem) .... etc

[https://tailwindcss.com/docs/padding](url)

Reply:
done